src/store/segmentGroups.ts

```diff
@@ -27,6 +33,36 @@ import {
   selectionEquals,
 } from './datasets';
 
+const readImage = async (file: File) => {
+  if (file.name.endsWith('.vti'))
+    return (await vtiReader(file)) as vtkImageData;
+
+  const { image } = await readImageItk(null, file);
+  return vtkITKHelper.convertItkToVtkImage(image);
+};
+
+const writeImage = async (format: string, image: vtkImageData) => {
+  if (format === 'vti') {
+    return vtiWriter(image);
+  }
+  // copyImage so writeImage does not detach live data when passing to worker
+  const itkImage = copyImage(vtkITKHelper.convertVtkToItkImage(image));
+
+  // Transpose the direction matrix to fix bug in @itk-wasm/image-io.writeImage
+  // Remove when @itk-wasm/image-io version is above 0.5.0 https://github.com/InsightSoftwareConsortium/itk-wasm/commit/ad9ca85eedc47c9d3444cf36859569c529886bde
+  const oldDirection = [...itkImage.direction];
+  const { dimension } = itkImage.imageType;
+  for (let idx = 0; idx < dimension; ++idx) {
+    for (let idy = 0; idy < dimension; ++idy) {
+      itkImage.direction[idx + idy * dimension] =
+        oldDirection[idy + idx * dimension];
+    }
+  }
```

Comment:
Need to remember to nuke this workaround after bumping @itk-wasm/image-io

Reply:
Can you add a TODO prefix to the comment (or submit an issue after this PR is merged)? That way it's easier to find TODOs that need addressing in the codebase.

Reply:
I'll submit a issue after this PR.